central_v2/src/pages/Confirmation.tsx

```diff
@@ -0,0 +1,207 @@
+import React, { useState, useRef } from 'react';
+import { useTheme, styled } from '@mui/material/styles';
+import { TextField, Box, Typography } from '@mui/material';
+import { APIClientsContext } from '../lib/context/APIClientsContext';
+import { useTSAPIClientsContext } from '../hooks/context/useAPIClientsContext';
+import { ButtonType } from '../components/button/ButtonModels';
+import CentralButton from "../components/button/Button";
+import RightOnLogo from "../images/RightOnLogo.png";
+
+// Styled components
+const OuterBody = styled(Box)(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    height: '100%',
+    boxSizing: 'border-box',
+    justifyContent: 'center',
+}));
+
+const InnerBody = styled(Box)(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '24px',
+}));
+
+const ImageContainer = styled(Box)(({ theme }) => ({
+    display: 'flex',
+    justifyContent: 'center',
+}));
+
+const VerifyText = styled(Typography)(({ theme }) => ({
+    color: '#02215F',
+    fontFamily: 'Poppins, sans-serif',
+    fontWeight: 700,
+    fontSize: '24px',
+    lineHeight: '36px',
+    textAlign: 'center',
+}));
+
+const EnterText = styled(Typography)(({ theme }) => ({
+    fontWeight: 400,
+    fontSize: '16px',
+    lineHeight: '20px',
+    fontFamily: 'Rubik, sans-serif',
+    color: '#02215F',
+}));
+
+const CodeandResendContainer = styled(Box)(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '8px',
+}));
+
+const UserCodeTextBoxesContainer = styled(Box)(({ theme }) => ({
+    display: 'flex',
+    justifyContent: 'center',
+    gap: '8px',
+}));
+
+const UserCodeTextBoxes = styled(TextField)(({ theme }) => ({
+    width: '40px',
+    height: '54px',
+    borderRadius: '8px',
+    border: '2px solid #FFFFFF',
+    color: '#FFFFFF',
+    textAlign: 'center',
+    input: {
+        textAlign: 'center',
+    },
+}));
+
+const ResendCodeText = styled(Typography)(({ theme }) => ({
+    fontFamily: 'Rubik, sans-serif',
+    fontWeight: 400,
+    fontSize: '16px',
+    color: '#02215F',
+    textDecoration: 'underline',
+    textAlign: 'center',
+}));
+
+const VerifyBox = styled(Box)(({ theme }) => ({
+    display: 'flex',
+    justifyContent: 'center',
+}));
+
+// Props interface
+interface ConfirmationProps {
+    schoolEmail?: string;
+    frontImage?: File | null;
+    backImage?: File | null;
+    handlerImageUpload?: (file: File) => Promise<any>;
+    password?: string
+}
+
+// Use function declaration for the component
+function Confirmation({ schoolEmail = '', frontImage, backImage, handlerImageUpload, password}: ConfirmationProps) {
```

Comment:
didn't know you could assign a value to a prop in the declaration here. Can you tell me more about the `schoolEmail = ''`? If it's always empty, do we need to pass it in as a prop?